contracts/facets/FleetsFacet.sol

```diff
@@ -267,7 +267,18 @@ contract FleetsFacet is Modifiers {
                 ),
             "AppStorage: Not owner"
         );
-
+        // require owner of toPlanetId == msg.sender || owner of toPlanetId == address(this)
```

Comment:
this check has to happen in the sendingTransaction, not afterwards

Reply:
resolved in https://github.com/GalaxyThrone/contracts/pull/10/commits/ce492b3227865dad0ec1eec50c1dc2c6d5726090